server/src/main/java/de/uftos/repositories/ucdl/UcdlRepositoryImpl.java

```diff
@@ -11,24 +17,63 @@
  */
 @Repository
 public class UcdlRepositoryImpl implements UcdlRepository {
+  private final File ucdlFile = new File("/app/ucdl/ucdl.yml");
+  private HashMap<String, ConstraintDefinitionDto> currentDefinitions = null;
 
   @Override
   public String getUcdl() {
-    return "";
+    try {
+      ucdlFile.createNewFile();
+      FileReader reader = new FileReader(ucdlFile);
+      int readInformation = reader.read();
+      StringBuilder sb = new StringBuilder();
+      while (readInformation >= 0) {
+        sb.append((char) readInformation);
+        readInformation = reader.read();
+      }
+      return sb.toString();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
Could we use the more modern implementation from the `nio` package?

Reply:
Sure